src/Compilers/Core/Portable/DiagnosticAnalyzer/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -128,59 +129,41 @@ private void DeleteLeftoverDirectories()
             }
         }
 
-        protected override string PreparePathToLoad(string originalFullPath)
+        protected override string PreparePathToLoad(string originalAnalyzerPath, ImmutableHashSet<string> cultureNames)
```

Comment:
For this file I recommend just looking at the new code, not the diff. The code is pretty straight forward.